docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing to "project"
+
+The main purpose of this project is "objective"
+
+## Summary
+
+* [Code of Conduct](#code-of-conduct)
+* [Issues](#Issues)
+* [Pull Requests](#pull-requests)
+* [Git workflow](#git-workflow)
+* [Join the Project](#join-us)
+
+## Code of Conduct
+
+By participating and contributing to this project, you agree to uphold these rules:
+
+- Give credit where credit is due
+- Be respectful to all project participants
+- Follow the `Code of Conduct` from [contributor-covenant.org](https://www.contributor-covenant.org/version/1/4/code-of-conduct/)
```

Comment:
The [Contributor Covenant](https://www.contributor-covenant.org/) is really cool! This is definitely something we should adhere to :+1: